assets/examples/web-components/all-charts.html

```diff
@@ -0,0 +1,416 @@
+<html>
+  <head>
+    <!-- https://gist.github.com/dwnoble/822ce6018bb41113c866d703760c1def -->
```

Comment:
delete the comment?

Reply:
Removed here and in all other example html files.